src/app/games/game-summaries/game-summaries.component.scss

```
@@ -2,8 +2,19 @@ div {
   margin-right: -1em;
 }
 
+h2 {
+  margin: auto;
+  width: max-content;
+}
+
 game-summary {
   flex-grow: 0;
   margin-bottom: 12px;
   margin-right: 1em;
 }
+
+external-game {
+  flex-grow: 0;
+  margin-bottom: 12px;
+  margin-right: 1em;
+}
```

```suggestion
game-summary, external-game {
  flex-grow: 0;
  margin-bottom: 12px;
  margin-right: 1em;
}
```
Don't repeat styles like this when it can be avoided